program/verilator-verification.ppl.md

```diff
@@ -0,0 +1,33 @@
+# OpenHW Preliminary Project Proposal: Verilator verification for CORE-V MCU SoC
+
+
+[[_TOC_]]
+
+## SUMMARY OF PROJECT
+This project aims to provide a stable and reasonably reliable Verilator verification flow:
+a. capable of verifuing the CORE-V MCU SoC RTL 
+b. capable of supporting a SW developmetn flow that would yield an RTOS booting on an FPGA implementation of the CORE-V MCU SoC . 
+
+Verilator is a free and open-source software tool capable of converting Synthesizable Verilog/SystemVerilog to a cycle-accurate cycle-accurate, 2-state behavioral model in C++ or SystemC.
+
+##  COMPONENTS
+The project has two parts:
+
+### Component 1 - Verification plan(s) and documentation
+Including verification strategy, clear documentation of what is not verified with Verilator but instead done with commercial verification tools.
+
+### Component 2 - Verification environment
+Including clear documentation of verification tool chain, automation scripts and alignment with the CORE-V MCU SoC RTL (upstream) and the SW tool chain (downstream)
+
```

Comment:
### Component 3 - Standalone software modeling
Creation of a standalone program taking a software program as argument, running to completion, and returning the return code of the program.

### Component 4 - software debut target
Creation of a component usable as either a standalone program or library implementing the remote serial protocol for connection by GDB or LLDB. The library version would be suitable for integration within an IDE